stable/rocketchat/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+name: rocketchat
+version: 0.0.1
+description: Prepare to take off with the ultimate chat platform, experience the next level of team communications
+keywords:
+- chat
+- communication
+- http
+- web
+- application
+- nodejs
+- javascript
+- meteor
+home: https://rocket.chat/
+icon: https://cdn-www.rocket.chat/images/logo/logo.svg
+sources:
+- https://github.com/RocketChat/Docker.Official.Image/
+maintainers:
+- name: RocketChat
```

Comment:
@pierreozoux I can provide a list from Rocket.Chat that would handle any issues that would arise as well as of course your self.